testing/test-utils/src/lib/utils/path.ts

```diff
@@ -0,0 +1,3 @@
+export function toNormalizedPath(path: string): string {
+  return path.replace(/\\/g, '/');
+}
```

Comment:
There's already a [`toUnixPath`](https://github.com/code-pushup/cli/blob/main/packages/utils/src/lib/transform.ts#L112-L114) function which does this.

Reply:
Im very aware of it. But if I don't duplicate,then I cant use it in tests as it would create a cyclic deps

Reply:
I see :grimacing: OK, thanks for explaining :+1: